subsys/fs/nvs/README.md

```diff
@@ -0,0 +1,163 @@
+# Non volatile storage (NVS) for zephyr
+
+# Overview
+
+Storage of elements in flash in a FIFO manner. Elements are stored as
+identifier (id) - data pairs in a flash circular buffer. The flash area is
+divided into sectors. Elements are appended to a sector until storage space in
+the sector is exhausted. Then a new sector in the flash area is prepared for
+use (erased). Before erasing the sector it is checked that identifier - data
+pairs exist in the sectors in use, if not the id-data pair is copied.
```

Comment:
copied to where - to a scratch area? Are they rewriter back to the new active page?

Reply:
@nvlsianpu The data is copied to the new active sector before the new write is done. NVS does not use a scratch area.